learn/quickstart/arrays_strings.md

```diff
@@ -149,3 +149,34 @@ program allocatable_string
 end program allocatable_string
 ```
 
+
+## Array of strings
+
+The length of all strings in the array must be the same. The length assigned is typically the length of the longest element.
+
+
+__Example:__ string array
+```fortran
+program string_array
+  implicit none
+  character(len=10), dimension(2) :: keys, vals
+
+  keys = [character(len=10)        :: "user", "dbname"]
+  vals = [character(len=10)        :: "ben", "motivation"]
+
+  call show(keys, vals)
+
+  contains
+
+    subroutine show(akeys, avals)
+      character(len=*), intent(in) :: akeys(:), avals(:)
+      integer                      :: i
+
+      do i = 1, size(akeys)
+        print *, akeys(i), ": ", avals(i)
+      end do
+
+    end subroutine show
```

Comment:
I thought the indentation of `contains` was unnecessary, let's keep indentation at a minimum for better readability on mobile.

```suggestion
contains

  subroutine show(akeys, avals)
    character(len=*), intent(in) :: akeys(:), avals(:)
    integer                      :: i

    do i = 1, size(akeys)
      print *, akeys(i), ": ", avals(i)
    end do

  end subroutine show
```